docs/kube-api-custom-ocp-versions.md

```diff
@@ -0,0 +1,94 @@
+# Custom OpenShift Versions (Dynamic z-stream Support)
+
+As part of [Hive Integration](https://github.com/openshift/assisted-service/blob/master/docs/Kube-API.md), a means to dynamically (without restarting the service) update an OCP version is required.
```

Comment:
Here also I don't think it's worth bragging about not requiring a service restart.